### PR TITLE
create-app: add `EntityLinksCard` to the System `EntityPage`

### PR DESCRIPTION
**Pull request by GoFightNguyen**
Signed-off-by: Jason Nguyen <jsn.dev@outlook.com>

## Hey, I just made a Pull Request!

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by GoFightNguyen** [comment]
[A little context from discord](https://discord.com/channels/687207715902193673/923144214580191282/1009219322675220580)
I figured this PR can remove any ambiguity as to why the System's page does not include links.

File: .changeset/young-trees-rescue.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+'@backstage/create-app': minor
+---
+
+Add `EntityLinksCard` to the system `EntityPage`.
```
Comment (jhaals) [inline .changeset/young-trees-rescue.md:5]:
```suggestion
Added `EntityLinksCard` to the system `EntityPage`.

For an existing installation where you want to display the links card for entity pages of kind `system` you should make the following adjustment to `packages/app/src/components/catalog/EntityPage.tsx`
```

File: .changeset/young-trees-rescue.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+'@backstage/create-app': minor
```
Comment (jhaals) [inline .changeset/young-trees-rescue.md:2]:
```suggestion
'@backstage/create-app': patch
```

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks for the PR! I have suggested some minor adjustments and pointers for existing users that would like to apply this change themselves.

**Comment by GoFightNguyen** [comment]
Thanks @jhaals, I applied these changes and squashed them into the first commit

**Review by jhaals — APPROVED** [review]
Nice work! Thanks 👍

**Pull request merged**